qiita_db/test/test_portal.py

```diff
@@ -0,0 +1,33 @@
+from unittest import TestCase, main
+
+from qiita_core.util import qiita_test_checker
+from qiita_db.portal import (
+    add_study_to_portal, remove_study_from_portal, get_studies_by_portal)
+from qiita_db.study import Study
+
+
+@qiita_test_checker()
```

Comment:
All the tests in here will fail if in the config file the portal is not defined as Qiita. Please, enclose the entire file in an if statement.